src/scripts/homebuttonpage.js

```diff
@@ -0,0 +1,17 @@
+function homebuttonpage() {
+  const content = document.querySelector('.content');
+  const homeButton = document.querySelector('.home');
+  homeButton.addEventListener('click', () => {
+    const contactContent = document.querySelector('.contact_content');
+    const menuContent = document.querySelector('.menu_content');
+    if (contactContent !== null) {
+      contactContent.parentNode.removeChild(contactContent);
+      document.body.appendChild(content);
+    }
+    if (menuContent !== null) {
+      menuContent.parentNode.removeChild(menuContent);
+      document.body.appendChild(content);
+    }
+  });
+}
+export default homebuttonpage();
```

Comment:
- [ ] This isn't the right way to export a module. Do this instead: `export default homebuttonpage;`. **Fix all other exported modules also.**